lib/global_uid.rb

```diff
@@ -27,3 +27,8 @@ class TimeoutException < Exception ; end
 if ActiveRecord::VERSION::MAJOR >= 4
   ActiveRecord::SchemaDumper.send(:prepend, GlobalUid::SchemaDumperExtension)
 end
+
+# Make sure that GlobalUID is disabled for ActiveRecord's Internal Metadata table
+if ActiveRecord::VERSION::MAJOR >= 5
+  ActiveRecord::InternalMetadata.global_uid_disabled
```

Comment:
I'd rather have it blow up saying the class does not exist then skipping it by accident